### PR TITLE
Support amp-playbuzz component (Issue - #643)

### PR DESCRIPTION
**Pull request by lysk88**
Fixes issue - #643

### CODE REVIEW

File: includes/sanitizers/class-amp-playbuzz-sanitizer.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+/**
+ * Converts Playbuzz embed to <amp-playbuzz>
+ */
+class AMP_Playbuzz_Sanitizer extends AMP_Base_Sanitizer{
+
+    public static $tag = 'div';
+    public static $pb_class = 'pb_feed';
+
+
+
```
Comment (mjangda) [inline includes/sanitizers/class-amp-playbuzz-sanitizer.php]: Can we please remove the extra linebreaks here and through the rest of the file?

File: includes/sanitizers/class-amp-playbuzz-sanitizer.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            if(self::$pb_class !== $node -> getAttribute('class')){
+                continue;
+            }
+
+            $old_attributes = AMP_DOM_Utils::get_node_attributes_as_assoc_array( $node );
+
+            $new_attributes = $this->filter_attributes( $old_attributes );
+
+
+            if(!isset($new_attributes['data-item']) && !isset($new_attributes['src'])){
```
Comment (mjangda) [inline includes/sanitizers/class-amp-playbuzz-sanitizer.php]: Can we please following spacing standards here and through the rest of the file? https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage

Comment (lysk88) [inline includes/sanitizers/class-amp-playbuzz-sanitizer.php]: @mjangda  - Should i work against WordPress or WordPressVIP coding standards?

Comment (mjangda) [inline includes/sanitizers/class-amp-playbuzz-sanitizer.php]: WordPress Coding Standards please.

Comment (lysk88) [inline includes/sanitizers/class-amp-playbuzz-sanitizer.php]: Will do, thanks!

**Comment by lysk88** [comment]
@mjangda - fixed, please review, thanks!

**Pull request merged**